content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -174,6 +174,41 @@ spec:
 
 However, the particular path specified in the custom recycler Pod template in the `volumes` part is replaced with the particular path of the volume that is being recycled.
 
+### Pre-binding
+
+The control plane can [bind Persistent Volume Claims to matching Persistent Volumes](#binding) in the
+cluster. However, if you want a Persistent Volume Claim to bind to a specific Persistent Volume, you need to pre-bind them.
+
+You can specify the Persistent Volume in your Persistent Volume Claim, then the Persistent Volume Claim will be able to bind to this specific Persistent Volume. If the Persistent Volume exists and is not reserved for other Persistent Volume Claims (has Persistent Volume Claim specified in `claimRef` field), the Persistent Volume and Persistent Volume Claim will be bound regardless of whether the Persistent Volume satisfies some matching criteria, e.g. node affinity. Storage class, access modes, and requested storage size are still checked.
+
+```
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: foo-pvc
+  namespace: foo
+spec:
+  volumeName: foo-pv
+  ...
+```
+
+This method does not guarantee any binding privileges to the Persistent Volume. If your specified Persistent Volume can be used by other Persistent Volume Claims, you need to reserve it first by specifying the Persistent Volume Claim in the Persistent Volume using the `claimRef` field so that other Persistent Volume Claims can not bind to it.
```

Comment:
```suggestion
This method does not guarantee any binding privileges to the PersistentVolume. If other PersistentVolumeClaims could use the PV that you specify, you first need to reserve that storage volume. List the relevant PersistentVolumeClaim in the `claimRef` field of the PV so that other PVCs can not bind to it.
```